src/frontend/src/features/utils/task-import-parser/index.ts

```
@@ -0,0 +1,46 @@
+        const value = localStorage.getItem(this.#key);
+        if (!value) return null
+
+        const { taskId, expiresAt } = this.#deserialize(value);
+        if (Date.now() <= expiresAt) return taskId;
+
+        this.remove();
+        return null;
+    }
+
+    set(taskId: string) {
+        if (typeof window === undefined || !this.mailboxId) return;
```

Biome (2.1.2)

[error] Invalid typeof comparison.

Compare with one of the following string literals:


(lint/correctness/useValidTypeof)